packages/kitchen/src/pan/Hover.vue

```diff
@@ -8,9 +8,11 @@
       </core-title>
       <core-section>
         <v-hover>
-          <v-sheet slot-scope="{ hover }">
-            {{ hover }}
-          </v-sheet>
+          <template #default="{ hover }">
```

Comment:
`<v-hover #default="{hover}">` (similar thing can be done also in other components)

Reply:
@jacekkarczmarczyk `v-hover`? maybe `v-sheet`?

Reply:
no, v-hover: https://codepen.io/anon/pen/QYzWdg

Reply:
will do